app/models/server/models/LivechatBusinessHours.ts

```diff
@@ -0,0 +1,36 @@
+import moment from 'moment';
+import { ObjectId } from 'mongodb';
+
+import { Base } from './_Base';
+import { LivechatBussinessHourTypes } from '../../../../definition/ILivechatBusinessHour';
+
+export class LivechatBusinessHours extends Base {
+	constructor() {
+		super('livechat_business_hours');
+
+		if (this.find({ type: LivechatBussinessHourTypes.SINGLE }).count() === 0) {
+			this.insert({
+				_id: new ObjectId().toHexString(),
+				name: '',
+				active: true,
+				type: LivechatBussinessHourTypes.SINGLE,
+				ts: new Date(),
+				workHours: [
+					{ day: 'Monday', start: '08:00', finish: '20:00', code: 1, open: true },
```

Comment:
It could be a loop, right?
Also, apart from the first one, all of the others have the `code = 2`, is that correct?

I think this could be a helper function.